File: 1-Iniciante/1041.py
X, Y = [float(x) for x in input().split()]

if X == 0.0 or Y == 0.0:
    if X == 0.0 and Y == 0.0: print('Origem')
    elif X == 0.0: print('Eixo Y')
    else: print('Eixo X')
else:
    if (X > 0.0):
        if (Y > 0.0): print('Q1')
        else: print('Q4')
    else:
        if (Y > 0.0): print('Q2')
        else: print('Q3')

File: 1-iniciante/1041.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(void)
{
    float X, Y;

    cin >> X >> Y;

    if (X == 0.0 || Y == 0.0) {
        if (X == 0.0 && Y == 0.0) cout << "Origem" << endl;
        else if (X == 0.0) cout << "Eixo Y" << endl;
        else cout << "Eixo X" << endl;
    }
    else {
        if (X > 0.0) {
            if (Y > 0.0) cout << "Q1" << endl;
            else cout << "Q4" << endl;
        }
        else {
            if (Y > 0.0) cout << "Q2" << endl;
            else cout << "Q3" << endl;
        }
    }
    return 0;
}
